Assets/Scripts/TES/MorrowindEngine.cs

```diff
@@ -285,33 +286,33 @@ public void Update()
 		}
 		public void CastInteractRay()
 		{
-			interactTextObj.SetActive(false);
-
 			// Cast a ray to see what the camera is looking at.
-			var ray = new Ray(playerCameraObj.transform.position, playerCameraObj.transform.forward);
+			Ray ray = new Ray(playerCameraObj.transform.position, playerCameraObj.transform.forward);
```

Comment:
I generally use "var" if the type is clear from the initial value. This is a minor style preference, so I don't mind if others prefer to avoid "var", but please don't Ctrl+F all the "var"s and replace them with types! They can cut down on verbosity sometimes.